### PR TITLE
OP-1130 improve JPA methods - dicomtype package

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-1130

dicomtype package updates to match Core PR: https://github.com/informatici/openhospital-core/pull/1108

### CODE REVIEW

File: src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java [hunk 1]
```diff
@@ -198,37 +197,24 @@ private JButton getOkButton() {
...
-							MessageDialog.error(null, "angal.common.datacouldnotbesaved.msg");
-						} else {
-							dispose();
-						}
+						dicomTypeBrowserManager.newDicomType(dicomType);
+						fireDicomTypeInserted(dicomType);
+						dispose();
 					} catch (OHServiceException e1) {
-						OHServiceExceptionUtil.showMessages(e1, DicomTypeEdit.this);
+						OHServiceExceptionUtil.showMessages(e1, this);
```
Comment (mwithi) [inline src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java:206]: Did you test this? I recall the owner `this` was not enough, but maybe now it is different

Comment (dbmalkovsky) [reply src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java:206]: Yes I tested it and it works fine.

Comment (mwithi) [reply src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java:206]: Ok, FWIW there are plenty of these all around GUI

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**